eq-author-api/schema/resolvers/base.js

```diff
@@ -161,6 +162,16 @@ const Resolvers = {
     },
     me: (root, args, ctx) => ctx.user,
     users: () => listUsers(),
+    triggerPublish: async (root, { questionnaireId }) => {
+      const result = await fetch(
+        `${process.env.SURVEY_REGISTER_URL}${questionnaireId}`
```

Comment:
I think this is fine as a first pass, but I think when we look at adding `eq_id` and `form_type` in the next story, this should be changed to POST to the registry instead. It's more semantically correct. The use of HTTP GET verb implies that we're querying, when actually we're adding something to the registry.

Reply:
Yeah you're right it really should be a fetch, it's a pretty small change both here and in the register so I'll just add it here.